lambdas/utils/get_aws_region.py

```diff
@@ -0,0 +1,8 @@
+import os
+
+
+DEFAULT_REGION = "eu-west-2"
+
+
+def get_aws_region() -> str:
+    return os.environ.get("AWS_REGION", DEFAULT_REGION)
```

Comment:
This function is not necessary, it's covered by the `set_env` function in the global conftest.py

Reply:
Done